docs/tutorial/part-one/index.md

```diff
@@ -187,12 +187,11 @@ Make the second page component look something like:
 import React from "react";
 import Link from "gatsby-link";
 
-export default () => (
+export default () =>
```

Comment:
This is the reason I started updating this doc. These parentheses seem unnecessary and they are not in the first example.  I thought it might confuse someone totally new to React and Gatsby.

Reply:
We started running Prettier on the markdown code examples recently which perhaps added this? I'll pull your branch locally and run Prettier again and see what happens.

Reply:
Hmm it did add it back... investigating.

Reply:
Yeah, it's because the other code blocks earlier have line highlighting e.g. `jsx{1-2}` which makes Prettier skip over them. We'll be moving soon-ish to an inlined way to mark lines to be highlighted which will make Prettier work more consistently.

Reply:
Okay. Thanks for looking into this. I noticed it and really wondered what was up.